PackagingProjects/BuildAgentPayload/BuildAgentPayload.csproj

```diff
@@ -13,6 +13,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <SonarQubeExclude>True</SonarQubeExclude>
```

Comment:
I've excluded the two packaging projects from analysis since they don't contain any code, just packaging scripts.